Allow other keypresses in LogListWidget

NotTsunami opened the pull request:
By not emitting the native signal, all other keypresses other than
the copy and select keysequences are ignored. This should reallow
keyboard navigation within LogListWidget objects.

Closes #12172.

NotTsunami commented:
Currently untested but I'm 99% sure this is why keyboard navigation didn't work. Will test on Windows tonight!

jagannatharjun commented:
xD, I got you https://github.com/qbittorrent/qBittorrent/pull/12144/commits/4175e8d776ee7ad542d45af2f84d403b44ca08d0

NotTsunami commented:
> xD, I got you [4175e8d](https://github.com/qbittorrent/qBittorrent/commit/4175e8d776ee7ad542d45af2f84d403b44ca08d0)

LOL

NotTsunami commented:
@jagannatharjun Well, that confirms my guess is right. This PR should still fix the execution log since it inherits LogListWidget, but my questions to you then is: Is SelectAll one of the natively handled keysequences?

jagannatharjun commented:
> Is SelectAll one of the natively handled keysequences?

It works on my end with QListView, so I removed it and afaik Qt works similar across the platform.

NotTsunami commented:
> > Is SelectAll one of the natively handled keysequences?
> 
> It works on my end with QListView, so I removed it and afaik Qt works similar across the platform.

Okay, cool. I'm not sure what the milestone for your PR is, so it's up to @Chocobo1 on whether or not this PR should continue.

NotTsunami commented:
https://github.com/qt/qtbase/blob/dev/src/widgets/itemviews/qabstractitemview.cpp#L2485

It looks like it is handled, and copy is too. I'm not sure why we overrode this in the first place then. Might be able to remove the function from all 3 locations?

jagannatharjun commented:
Copy will only copy the last selected row, we select the text from all the selected rows

Chocobo1 commented:
>Okay, cool. I'm not sure what the milestone for your PR is, so it's up to @Chocobo1 on whether or not this PR should continue.

IMO that PR still has much for review :\ It will definitely be faster to get things done in a leaner PR and I think this PR is beneficial for the next release.

NotTsunami commented:
> > Okay, cool. I'm not sure what the milestone for your PR is, so it's up to @Chocobo1 on whether or not this PR should continue.
> 
> IMO that PR still has much for review :\ It will definitely be faster to get things done in a leaner PR and I think this PR is beneficial for the next release.

Okay. I updated the commit description to fit qBit guidelines and removed the override for QKeySequence::SelectAll.

Chocobo1 commented:
@NotTsunami 
Thank you!
